jest.config.ts

```diff
@@ -35,19 +33,20 @@ const config: Config.InitialOptions = {
 			branches: 80,
 			functions: 80,
 			lines: 80,
-			statements: -7,
+			statements: -11,
```

Comment:
Why?

Reply:
After Update jest to v29 are now 11 uncovered tests. But Team NorthStar has ticket for that [BC-2536](https://ticketsystem.dbildungscloud.de/browse/BC-2536). I will communicate it.